modules/tensor_mechanics/doc/content/documentation/modules/tensor_mechanics/generalized_plane_strain.md

```diff
@@ -27,7 +31,7 @@ The kinematical equations of the generalized plane strain problem are identical
 \end{matrix}\right\}
 \end{equation}
 
-The equilibrium equations for the generalized plane strain problem in the $x$-$y$ plane are given as
+The equilibrium equations for the generalized plane strain problem in the are given as
```

Comment:
I think I'd leave this how it was, but if you want to change this, drop "in the".